docs/usage/README.md

```diff
@@ -1,3 +1,96 @@
 # Usage Guides
 
-Place for the `cephlet` usage guides.
+This section provides an overview on how `Volume`s from the [onmetal-api](https://github.com/onmetal/onmetal-api) project can be provisioned using the `cephlet` provider.
+
+## Available Pools and Classes
+As a user you can request storage by creating a `Volume`. It will be allocated in the referenced `VolumePool`. 
+The `VolumeClasses` define the capabilities in terms of IOPS, BPS limits and other resource requirements. 
+
+Get the available `VolumePools` with the corresponding `VolumeClasses`
+```shell
+kubectl get volumeclasses 
+NAME   AGE
+fast   4d18h
+slow   4d18h
+
+kubectl get volumepool
+NAME   VOLUMECLASSES   AGE
+ceph   fast,slow       4d17h
+```
+
+## Creating a `Volume`
+A `Volume` is referencing a `VolumePool` and a matching `VolumeClass` which the `VolumePool` supports.
+
+```yaml
+# sample-volume.yaml
+apiVersion: storage.api.onmetal.de/v1alpha1
+kind: Volume
+metadata:
+  name: sample-volume
```

Comment:
To keep things consistent I would keep the `namespace: default` here as it is also specified in the other examples.